data/net/minecraft/world/level/levelgen/feature/featuresize/FeatureSize.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/world/level/levelgen/feature/featuresize/FeatureSize
+	METHOD <init> (Ljava/util/OptionalInt;)V
+		ARG 1 minClippedHeight
+	METHOD getSizeAtHeight (II)I
+		ARG 1 x
+		ARG 2 z
```

Comment:
I had difficulty finding good names for these. In the case of trees he first arg is always the tree height. The second arg is a moving height between 0 and height.
```suggestion
		ARG 1 height
		ARG 2 midpoint
```